coloringbook/admin/utilities.py

```diff
@@ -1,3 +1,5 @@
+# coding=utf-8
```

Comment:
This magic comment appears to be necessary. My container will not run without it.

![image](https://github.com/UUDigitalHumanitieslab/coloringbook/assets/73882506/16375137-f4a5-421c-811a-083121211bb6)

(The character denoted with \xc3 is ã, apparently.)